website/documentation/050-tutorials/content/howto/shell-to-node/_index.md

```diff
@@ -0,0 +1,333 @@
+
+For this scenario, CloudProvider typically have extensive documentation (e.g [AWS](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstances.html) & [GCP](https://cloud.google.com/compute/docs/instances/connecting-to-instance) and in [some cases tooling support](https://cloud.google.com/compute/docs/instances/connecting-advanced#vpn)). 
+However, these approaches are mostly CloudProvider specific and require interaction via the CloudProvider CLI or API. 
+
+Instead you can use `gardenctl` that provides out-of-the-box support to ssh into an instance in a private subnet for all major Cloud Providers.
+
+## Identifying the problematic instance
+Firstly we need to pinpoint which instance is having problems. In Gardener, worker pools can be created in different CloudProvider regions, zones and accounts. So identifying the problematic instance via the CloudProvider is the first stepping-stone. 
+Even though having identified the correct CLoudProvider region and account of the Shoot Cluster, the instance would typically show up as sucessfully started / running in the CloudProvider dashboard / API - so it is not immediately obvious which instance has a problem.
+
+Instead we can use the Gardener API / CRDs to obtain the instance identifier in a Cloud-agnostic way.
+Gardener uses the [Machine Controller Manager](https://github.com/gardener/machine-controller-manager) to create the Shoot worker nodes. 
+For each worker node, the Machine Controller Manager creates a `Machine` CRD in the Shoot namespace in the respective Seed cluster. 
+Usually the problematic instance can be identified as the respective `Machine` CRD is `pending`.
+The `Machine` already contains the instance / node name in the .status, obtain it via:
+`k get machine <machine-name> -o json | jq -r .status.node`
+
+This is all the information needed to go ahead and use `gardenctl ssh` to get a shell to the node (you can skip to the section [Gardenctl ssh](#gardenctl-ssh) below). 
+
+Additionally we can identify the Cloud Provider, the Cloud Provider specific identfier of the instance and the region via: 
+
+`
+k get machine <machine-name> -o json | jq -r .spec.providerID // e.g aws:///eu-north-1/i-069733c435bdb4640
+`
+In this case: Cloud Provider `aws` with the ec2 instance-id `i-069733c435bdb4640` in region `eu-north-1`.
+Additonal information about the instance can be found in the associated `<AWS/GCP/..>MachineClass` and can be obtained via the Cloud Provider specific dashboard & CLI.
+
+
+## Gardenctl ssh
+
+Using the node name of the problematic instance, we can use the `gardenctl ssh` command to get a shell to the Cloud Provider instance via an automatically set up bastion.
+The steps that `gardenctl` performs for each Cloud Provider [can be found here]( https://github.com/gardener/gardenctl/blob/10a537942b94234914758c0f6d053dc1cf218ecd/pkg/cmd/ssh_aws.go#L53-L52).
+
+
+
+Use the following commands:
```

Comment:
Same comment as for using gadenctl with an operational node - split and consider adding a few words to lower the entry level particularly as far as gardenctl is concerned.